packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -20,6 +20,60 @@ const uniqueRoute = (route) => {
   return `${route}-${routeCount}`
 }
 
+describe('network stubbing - not skipped', () => {
+  it('stops waiting when an xhr request is canceled', () => {
+    cy.visit('http://localhost:3500/fixtures/generic.html')
+
+    cy.intercept('POST', /users/, {
+      body: { name: 'b' },
+      delay: 2000,
+    }).as('createUser')
+
+    cy.window()
+    .then((win) => {
+      const xhr = new win.XMLHttpRequest()
+
+      xhr.open('POST', '/users/')
+
+      xhr.send()
```

Comment:
would possibly adding [abort()](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/abort) after this help make this always deterministic?

Reply:
Maybe? I wonder if using `abort()` would cause this test to display the same issues that the fetch test does.